test/check-application

```diff
@@ -77,6 +77,47 @@ class TestApplication(testlib.MachineCase):
         else:
             return self.admin_s.execute(cmd)
 
+    def waitPodRow(self, podName):
+        self.browser.wait_present("#table-" + podName)
+
+    def waitPodContainer(self, podName, containerList):
+        print(containerList)
+        if len(containerList):
+            for container in containerList:
+                self.check_container(container["id"], True, [container["name"], container["image"], container["command"], container["state"]])
```

Comment:
This does not check that the container is associated with the pod, just that it is displayed.